README.md

```diff
@@ -556,6 +562,80 @@ Specifies any additional attributes to add to the Valve. Should be a hash of the
 
 Specifies any attributes to remove from the Valve. Should be a hash of the format 'attribute' => 'value'. This parameter is optional.
 
+####tomcat::config::context
+
+#####`$catalina_base`
+
+Specifies the root of the Tomcat installation.
+
+####tomcat::config::context::resource
+
+#####`$catalina_base`
+
+Specifies the root of the Tomcat installation.
+
+#####`$resource_name`
+
+is the name of the Resource to be created, relative to the java:comp/env context.
+
+#####`$auth`
+
+authentication type for the Resource.
+
+#####`$closeMethod`
```

Comment:
Should be fixed, but I found another camel case in globalnamingresources.pp that I accidentally fixed and reverted, since it should probably be better suited for a nother pull